addons/mail/static/src/new/public/discuss_public.xml

```diff
@@ -2,8 +2,11 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.discuss_public" owl="1">
-    <div>
-        DISCUSS PUBLIC
+    <div class="o-discuss-public d-flex flex-column h-100 w-100 position-absolute" t-attf-class="{{ className }}" t-ref="root">
+        <DiscussPublicWelcomePage
+            channel="props.data.channelData"
+            shouldAddGuestAsMemberOnJoin="props.data.discussPublicViewData.shouldAddGuestAsMemberOnJoin"
```

Comment:
`shouldAddGuestAsMemberOnJoin` => `addSelfMemberOnJoin` (shorter and a bit easier to read)